server/src/routers/router.es6

```diff
@@ -12,18 +15,31 @@ const specialUseRouter = require('./specialuse.es6');
 
 const router = express.Router();
 
+let origin = 'http://localhost:4200';
+if (util.isProduction()) {
+  origin = vcapConstants.INTAKE_CLIENT_BASE_URL;
+}
+const corsOpts = {
+  origin,
+  optionsSuccessStatus: 200,
+  credentials: true,
+  allowedHeaders: 'accept, content-type',
+  methods: 'GET, PUT, POST, DELETE, OPTIONS, PATCH'
+};
+
 /** Allow any server to check the preflights. */
-router.options('*', middleware.setCorsHeaders, (_req, res) => {
+router.options('*', cors(corsOpts), (_req, res) => {
   res.set('Access-Control-Allow-Headers', 'accept, content-type');
   res.set('Access-Control-Allow-Methods', 'GET, PUT, POST, DELETE, OPTIONS, PATCH');
+  res.set('Cache-Control', 'no-cache');
```

Comment:
We want the cache header to apply to ALL of the requests, by placing it here, it only applies to the `OPTIONS` requests. I think it should be added at the route, for all requests, along the lines of the sample code I sent you earlier.

Reply:
Should be all set now, thanks David!